src/index.js

```diff
@@ -28,6 +29,8 @@ function closeSocket() {
 
 const store = configureStore({ webSocket });
 
+store.dispatch(setLocale(navigator.language.split(/[-_]/)[0]));
```

Comment:
we'll need multiple segments (e.g. pt_BR, zh_TW, etc.), and we should also make some effort to provide reasonable fallbacks where appropriate or at least follow the user's preferences in `navigator.languages` (plural) where supported.

For now let's just store `navigator.language`, we should fallback to 'en' if not supported.